docker/README.md

```diff
@@ -0,0 +1,43 @@
+# Docker for Compiling and Running HEIR
+
+Docker can also be used to compile and run HEIR project from the bleeding edge source on git. This involves building a Docker image of the development environment and trigger the build which follows below.
```

Comment:
These lines are still not wrapped properly for mdformat. Are you sure you pushed your pre-commit changes?

I recommend setting it up as a git pre-commit hook: fa37f47930ff22bab03b8ee20f6ec3fe540739c4